### DISCUSSION TITLE
month/

### DISCUSSION THREAD

**Discussion by giscus** [OweQian/oweqian.github.io#11] [General]
# month/

2023 2023-02 02-01 02-02 Damn cool!!! 02-03 node.js 还是挺有意思的。 02-04 😠 无知和过度的商业宣传形成了健康问题必然危机生命的大众认知，从维持生命的人性底层上滋生了心理层面的恐慌，驱

https://oweqian.xyz/month/

**Comment by EricZZZ** [comment]
居然没有老麦克和索尔🙄

**Reply by OweQian** [reply]
哈哈，等追完风骚律师再来摆老麦克和索尔。

**Comment by EricZZZ** [comment]
你终于追完“初恋”了，我还以为你忘了呢😭

**Reply by OweQian** [reply]
😀都记在小本本上了，一个一个来。

**Comment by EricZZZ** [comment]
嘿嘿嘿~小猫咪🐱

**Comment by EricZZZ** [comment]
https://chat-ai.logcg.com/ 试试这个，除了比官网贵1.5倍，其他都挺好 😐

**Reply by OweQian** [reply]
![b8961254ef1f73b6b3c3482b299ae70](https://user-images.githubusercontent.com/25632596/231093675-e69dcda7-1fd6-40c9-bd3d-1e8f9ee61848.png)
🙈不会像这个人一样被查吧。。

**Reply by EricZZZ** [reply]
应该不会吧，境外的，刚又冲了9块钱的 😥

**Reply by OweQian** [reply]
https://chat.theb.ai 这个号称现在有10万个tg bot用户和100万个网站用户，说是免费的，还不晓得要怎么割呢

**Reply by EricZZZ** [reply]
666，免费的

**Comment by EricZZZ** [comment]
管好钱，先从记账开始，推荐使用 [Beancount](https://sspai.com/post/59777)

**Reply by OweQian** [reply]
😀 收，谢谢老板。

**Comment by EricZZZ** [comment]
不“嘿嘿嘿~小猫咪了”？😂

**Comment by EricZZZ** [comment]
小心曹姓大佬拿起法律武器，制裁你🤪

**Reply by OweQian** [reply]
🤪 哈哈，让我这个小研发没有好果汁吃。

**Comment by EricZZZ** [comment]
我这次得新冠，十有八九就是发版闹得，天天发版，影响睡眠，让病毒有了可乘之机，你小心点

**Reply by OweQian** [reply]
😭 应该就是这个闹得，今天开始头晕，看不清代码，都怀疑二阳了。

**Comment by EricZZZ** [comment]
催更😜

**Comment by Lemonade1172** [comment]
哇塞，你这个碎碎念持续记录的时间可真不短了
